packages/builder/src/components/automation/SetupPanel/RowSelector.svelte

```diff
@@ -43,6 +43,11 @@
   }
 
   const coerce = (value, type) => {
+    var re = new RegExp(/{{([^{].*?)}}/g)
```

Comment:
Does this need to be function scoped? Could probably just use `const` here.

Reply:
whoops, you're right!